src/vue/backend/src/components/Header.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <idx-navbar customClass="header" tag="header" type="light" theme="light">
+        <idx-navbar-brand customClass="mr-auto" link="/home">
+            <img src="@/assets/idx-logo.svg" height="30" alt="IDX Broker" loading="lazy">
+        </idx-navbar-brand>
+        <idx-block>
+            <idx-button
+                rounded
+                theme="success"
+                customClass="my-2 my-sm-0"
+                type="submit">
+                Hello Guest
+            </idx-button>
+        </idx-block>
+    </idx-navbar>
+</template>
+
+<script>
+export default {
+    name: 'Header',
+    data () {
+        return {
+            expanded: true
+        }
+    },
+    methods: {
+        horizontalExpand: function (e) {
+            this.expanded = !this.expanded
+        },
+        iconBarExpand: function (e) {
+            this.expanded = true
+        }
+    }
+}
+</script>
+
+<style scoped lang="scss">
+    @import '~@idxbrokerllc/idxstrap/dist/styles/globalVariables.scss';
```

Comment:
You should not need `globalVariables.scss` because it's loaded in globally in the vue config file: https://github.com/idxbroker/wordpress-plugin/blob/release/3.0.0/src/vue/backend/vue.config.js#L33

Additionally, since you're already importing the `base.scss` in `App.vue` (which this is a child of) you should not need to do it twice